bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -34,8 +35,12 @@ class ControlStateManager {
   void wedge() { wedged = true; }
   void unwedge() { wedged = false; }
   void markRemoveMetadata(bool include_st = true) { removeMetadataCbRegistry_.invokeAll(include_st); }
-  void setPruningProcess(bool onPruningProcess) { onPruningProcess_ = onPruningProcess; }
+  void setPruningProcess(bool onPruningProcess) {
```

Comment:
what happens if setPrunningProcess(true) is called multiple times in a row?

Reply:
right, I have added an assert